widget/menu_internal_test.go

```diff
@@ -45,12 +45,12 @@ func TestMenu_ItemTapped(t *testing.T) {
 	sm := mi3.child
 	smi := sm.Items[0].(*menuItem)
 	assert.Equal(t, subItem, smi.Item)
-	assert.True(t, sm.Visible(), "sub menu is visible")
+	assert.True(t, sm.Visible(), "submenu is visible")
 
 	test.Tap(smi)
 	assert.True(t, dismissed, "tap on sub item dismisses the root menu")
 	assert.True(t, m.Visible(), "tap on item does not hide the menu … OnDismiss is responsible for that")
-	assert.False(t, sm.Visible(), "tap on sub item hides the sub menu")
+	assert.False(t, sm.Visible(), "tap on sub item hides the submenu")
```

Comment:
this sentence includes “sub items” but “submenu” - I think it only appears twice (once 2 lines above), maybe “a child item” might clear up the slight inconsistency?